tests/tagged_ptr_test.cpp

```diff
@@ -100,18 +101,21 @@ int main(int argc, char *argv[]) {
             abort();
         }
 
+#if defined(__SANITIZE_ADDRESS__)
```

Comment:
Does `ENABLE_ASAN` not work here? Its already set by the Makefile

Reply:
it possibly can, but at least here we more certain if it comes from the compiler ?

Reply:
True but that probably means there should be a different check to verify that `address_sanitizer` and `memory_sanitizer` are present when ENABLE_ASAN is and that probably does not need to be here in a test.